pulsar-client-go/pulsar/logger.go

```diff
@@ -0,0 +1,48 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package pulsar
+
+import "fmt"
+
+type LoggerLevel int
+
+const (
+	DEBUG LoggerLevel = 0;
+	INFO  LoggerLevel = 1;
+	WARN  LoggerLevel = 2;
+	ERROR LoggerLevel = 3;
+)
+
+
+func (l LoggerLevel) String() string {
+	switch l {
+	case DEBUG:
+		return "DEBUG"
+	case INFO:
+		return "INFO "
+	case WARN:
+		return "WARN "
+	case ERROR:
+		return "ERROR"
+
+	default:
+		return fmt.Sprintf("%d", int(l))
```

Comment:
Not a big deal, but you can omit the `int()` wrap. I think it would better do `"UNKNOWN: %d"` or something so it is more obvious what is wrong.